<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgHideWifiEapTls</class>
 <widget class="QDialog" name="DlgHideWifiEapTls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to Hidden WLAN Network</string>
  </property>
  <widget class="QComboBox" name="cbxSecurity">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>185</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNetName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>145</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUserCertifyPwd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>505</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCaPwd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>385</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPwd">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>397</y>
     <width>18</width>
     <height>9</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbConn">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>80</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbNetName">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>150</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbDomain">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>310</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbCA">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>350</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxUserCertify">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>465</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="lbIdentity">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>270</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbUserCertifyPwd">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>510</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>660</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbUserKeyPwd">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>590</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxCA">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>345</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="lbUserPriKey">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>550</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbCaPwd">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>390</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDomain">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>305</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPwdSec">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>597</y>
     <width>18</width>
     <height>9</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIdentity">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>265</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUserKeyPwd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>585</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbUserCertify">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>470</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbAuth">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>230</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxAuth">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>225</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>660</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCA">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>425</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxConn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>75</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxUserPriKey">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>545</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="lbSecurity">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>190</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbBoder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>705</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbLeftupTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="lineUp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>125</y>
     <width>412</width>
     <height>1</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="lineDown">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>412</width>
     <height>1</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>lbBoder</zorder>
  <zorder>cbxSecurity</zorder>
  <zorder>leNetName</zorder>
  <zorder>leUserCertifyPwd</zorder>
  <zorder>leCaPwd</zorder>
  <zorder>checkBoxPwd</zorder>
  <zorder>lbConn</zorder>
  <zorder>lbNetName</zorder>
  <zorder>lbDomain</zorder>
  <zorder>lbCA</zorder>
  <zorder>cbxUserCertify</zorder>
  <zorder>lbIdentity</zorder>
  <zorder>lbUserCertifyPwd</zorder>
  <zorder>btnCancel</zorder>
  <zorder>lbUserKeyPwd</zorder>
  <zorder>cbxCA</zorder>
  <zorder>lbUserPriKey</zorder>
  <zorder>lbCaPwd</zorder>
  <zorder>leDomain</zorder>
  <zorder>leIdentity</zorder>
  <zorder>leUserKeyPwd</zorder>
  <zorder>lbUserCertify</zorder>
  <zorder>lbAuth</zorder>
  <zorder>cbxAuth</zorder>
  <zorder>btnConnect</zorder>
  <zorder>checkBoxCA</zorder>
  <zorder>cbxConn</zorder>
  <zorder>cbxUserPriKey</zorder>
  <zorder>lbSecurity</zorder>
  <zorder>lbLeftupTitle</zorder>
  <zorder>checkBoxPwdSec</zorder>
  <zorder>lineUp</zorder>
  <zorder>lineDown</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
